package.json

```diff
@@ -28,7 +28,8 @@
       "requireConfigFile": false
     },
     "rules": {
-      "no-undef": "error"
+      "no-undef": "error",
```

Comment:
This is already enabled in the PlayCanvas config: https://github.com/playcanvas/playcanvas-eslint-config/blob/main/index.js#L58

Reply:
thanks, removed